### PR TITLE
Add JSConf EU 2018 and fix typos on `endDate`

### PR DESCRIPTION
**Pull request by darahak**

### CODE REVIEW

File: conferences/2018/javascript.json [hunk 1]
```diff
@@ -150,6 +150,15 @@
     "twitter": "@fronttrends"
   },
   {
+    "name": "JSConf EU",
+    "url": "https://2018.jsconf.eu/",
+    "startDate": "2018-06-02",
+    "endDate": "2018-06-03",
+    "city": "Berlin",
+    "country": "Germany",
+    "twitter": "@jsconfeu",
```
Comment (nimzco) [inline conferences/2018/javascript.json]: You have to remove the coma here. `JSON` files don't support coma dangling. 😉

Comment (darahak) [reply conferences/2018/javascript.json]: Good catch, I just copy pasted from the template.
I also applied the fix there.

**Comment by prigara** [comment]
Thank  you!

File: conferences/2018/javascript.json (hunk 1 above)
Comment (nimzco) [reply conferences/2018/javascript.json]: Awesome thanks!

**Pull request merged**